uint8 row_i
uint8 num_coln
uint8 num_row
uint16 sensor0
uint16 sensor1
uint16 sensor2
uint16 sensor3
uint16 sensor4